docs/metrics/confusion-matrix.md

```diff
@@ -0,0 +1,203 @@
+every unique pair of actual and predicted labels. A multiclass classification confusion matrix with three classes
+would have the following format:
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| **Actual `Airplane`** | <span class="mg-cell-color-positive">Correct Prediction</span> | <span class="mg-cell-color-negative">Incorrect Prediction</span> | <span class="mg-cell-color-negative">Incorrect Prediction</span> |
+| **Actual `Boat`** | <span class="mg-cell-color-negative">Incorrect Prediction</span> | <span class="mg-cell-color-positive">Correct Prediction</span> | <span class="mg-cell-color-negative">Incorrect Prediction</span> |
+| **Actual `Car`** | <span class="mg-cell-color-negative">Incorrect Prediction</span> | <span class="mg-cell-color-negative">Incorrect Prediction</span> | <span class="mg-cell-color-positive">Correct Prediction</span> |
+
+And for example, if we are trying to calculate the counts of TP, FP, FN, and TN for class `Boat`:
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| **Actual `Airplane`** | <span class="mg-cell-color-positive">True Negative</span> | <span class="mg-cell-color-negative">False Positive</span> | <span class="mg-cell-color-positive">True Negative</span> |
+| **Actual `Boat`** | <span class="mg-cell-color-negative">False Negative</span> | <span class="mg-cell-color-positive">True Positive</span> | <span class="mg-cell-color-negative">False Negative</span> |
+| **Actual `Car`** | <span class="mg-cell-color-positive">True Negative</span> | <span class="mg-cell-color-negative">False Positive</span> | <span class="mg-cell-color-positive">True Negative</span> |
+
+
+**Example: Multiclass**
+
+Let's take a look at a multiclass classification example and plot a confusion matrix. In this example, we have three
+classes: `Airplane`, `Boat`, and `Car`. The multiclass classifier outputs the following inferences:
+
+<center>
+
+|  | Sample 1 | Sample 2 | Sample 3 | Sample 4 | Sample 5 | Sample 6 |
+| --- | --- | --- | --- | --- | --- | --- |
+| **Ground Truth** | `Airplane` | `Boat` | `Car` | `Airplane` | `Boat` | `Boat` |
+| **Inference** | `Airplane` | `Boat` | `Airplane` | `Airplane` | `Boat` | `Car` |
+
+</center>
+
+A confusion matrix for this example can be plotted:
+
+<center>
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| `Airplane` | 2 | 0 | 0 |
+| `Boat` | 0 | 2 | 1 |
+| `Car` | 1 | 0 | 0 |
+
+</center>
+
+In a different case, these counts may be much higher:
+
+<center>
+
+|  | Predicted `Airplane` | Predicted `Boat` | Predicted `Car` |
+| --- | --- | --- | --- |
+| `Airplane` | 200 | 0 | 0 |
+| `Boat` | 100 | 8,800 | 600 |
+| `Car` | 100 | 0 | 1,000 |
+
+</center>
+
+This confusion matrix reveals that a model is very good at identifying `Boat`: 8,800 of 9,500 `Boat` were correctly
```

Comment:
For readability we could:
...identifying the `Boat` class: x of y `Boat`s were...